packages/headless/src/product-listing.index.ts

```diff
@@ -35,9 +35,17 @@ export {
 } from './controllers/product-listing/pager/headless-product-listing-pager';
 
 export {
-  SortInitialState,
-  SortProps,
-  Sort,
-  SortState,
+  ProductListingSortInitialState,
+  ProductListingSortProps,
+  ProductListingSort,
+  ProductListingSortState,
   buildSort,
+  ProductListingSortBy,
+  ProductListingSortDirection,
+  ProductListingSortByRelevance,
+  ProductListingSortByFields,
+  ProductListingSortByFieldsFields,
+  ProductListingSortCriterion,
+  buildProductListingRelevanceSortCriterion,
+  buildProductListingFieldsSortCriterion,
```

Comment:
Should those be exported with `as SortBy`, `SortDirection`, without the prefix? (I'm not sure about the answer myself)

Reply:
I'm leaning more towards keeping them prefixed with `ProductListing`: I think it would make them easier to use (by developers using headless & developers coding headless) if they weren't prefixed, but some types/interfaces would clash with or be confused with the standard sort types/interfaces (`SortBy`/`SortBy`, `SortByRelevance`/`SortByRelevancy`, `SortDirection`/`SortOrder`, etc.).

Plus, prefixing makes the intended usage *really* obvious.

Reply:
An other solution which exists is aliasing the exported members and keeping the simple names internal to headless.

Something like: 
```
  SortBy as ProductListingSortBy,
  SortDirection as ProductListingSortDirection,
  SortByRelevance as ProductListingSortByRelevance
  // ...
```

Thoughts? @francoislg

Reply:
I like better the prefix for the internal code, this I'm sure :P